src/main/scala/midas/passes/fame/ChannelExcision.scala

```diff
@@ -56,7 +56,7 @@ class ChannelExcision extends Transform {
 
     // Step 1: Analysis -> build a map from reference targets to channel name
     state.annotations.collect({
-      case fta@ FAMEChannelConnectionAnnotation(name, WireChannel, Some(srcs), Some(sinks)) =>
+      case fta@ FAMEChannelConnectionAnnotation(name, PipeChannel(_), Some(srcs), Some(sinks)) =>
```

Comment:
If we're going to distinguish `fame.WireChannel` as a special thing with latency zero, should we change the name of the variables that contain all `PipeChannel`s?